resources/js/Pages/Admin/BookingRequests/CreateBookingRequestModal.vue

```diff
@@ -2,7 +2,7 @@
     <jet-dialog-modal :show="room" @close="closeModal" max-width="">
         <template #content>
 
-          <div class="overflow-y-auto h-96">
+          <div :dusk="'createBookingModal'" class="overflow-y-auto h-96">
```

Comment:
you can remove the binding and just use the string directly

Reply:
wasn't sure at first if i would need to generate an id number next to it at first.